### ISSUE TITLE
Unhelpful `AssertionError ''`after bisection

### ISSUE THREAD

**Issue by Quarub** [DeadCodeProductions/dead#35]
We seldomly get assertion errors after bisections; these have the empty string as error message which is not particularly helpful.
- [ ] Find and update the assert
- [ ] Investigate underlying issue    
```
INFO:root:Bisecting for approx. 9 steps
77df3be0dee415713cf5c79543f00532674f428b
988901ac74053408d2454f03651c8250ee88104c
INFO:root:Build log at [...]/logs/20220704-131628-llvm-83456f2b4fa2c12aadd9f8eea286affaf52a34ec.log
AssertionError: ''
```

**Comment by Quarub** [comment]
Further evidence of something being wrong:
```
INFO:root:Cache bisection: range size 32648 -> 45
INFO:root:Bisecting for approx. 6 steps
8b2f33231c2a9f774481aae57298420f7ade989f
1b1c8d83d3567a60280291c0adb95d1d60335509
INFO:root:Build log at [...]logs/20220726-211831-llvm-e7c8cd4a930465edd6c0ecbf3ad3119737988941.log
INFO:root:creduce logfile at [...]/logs/20220726-212211-creduce-836.log
```
Somewhere around case `226620`.

**Comment by Quarub** [comment]
There is a bug in `ccbuilder 0.0.5` where the `Repository` does not return the requested hash but prints it instead and returns `''`. 
This has been fixed in `0.0.6` so I assume that this issue is also fixed with it, as it looks awfully similar (empty string when requesting git hashes) and I haven't encountered it updating the dependency.
